test/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project (CBL_C_Tests)
 
 set(TOP ${PROJECT_SOURCE_DIR}/../)
 
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
+endif()
```

Comment:
A better way to handle this would be linking with the `Threads::Threads` module (you can see some examples of this elsewhere) in order to get the correct compiler flag.  This will do, though, since it is only for our internal use.

Reply:
I will work on this in a separate PR as it seems like the target_link_libraries setup in test's CMake doesn't seem right but work.